websites/O/OtakuEyes/metadata.json

```
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "297055773714808833",
+		"name": "Yù"
+	},
+	"service": "OtakuEyes",
+	"description": {
+		"en": "My own platform for watching animes",
```

Taken from your [GitHub ](https://github.com/YuToutCourt/OtakuEyes-Flask) project's README.
```suggestion
		"en": "OtakuEyes is a web application that allows you to watch anime online. OtakuEyes have the most differents anime sources.",
```